### PR TITLE
New version: SaaSGroup.Tower version 8.0.492

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#9639](https://github.com/SpecterShell/Dumplings/actions/runs/11893096702).

<details>

<summary>Log</summary>

````
Updated: 7.1.483 → 8.0.492
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/192165)

### CODE REVIEW

File: manifests/s/SaaSGroup/Tower/8.0.492/SaaSGroup.Tower.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  InstallerSha256: 53CE6F01C2B3BE0EE989BAB57DD7FE7C176D660DCD1E6185670B94E1163317AD
+  InstallModes:
+  - interactive
+  - silent
+  InstallerSwitches:
+    Silent: --silent
+    SilentWithProgress: --silent
+  ProductCode: Tower
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 8.0.492
```
Comment (mdanish-kh) [inline manifests/s/SaaSGroup/Tower/8.0.492/SaaSGroup.Tower.installer.yaml:25]:
```suggestion
```

Not needed, equal to PackageVersion

File: manifests/s/SaaSGroup/Tower/8.0.492/SaaSGroup.Tower.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://www.git-tower.com/apps/tower3-win/492-cb21208b/Tower-8.0.492.msi
+  InstallerSha256: 6172EC461833E6B84A6A8EF32DCCD1742648FD6950003742FB46A697B3933C63
+  InstallerSwitches:
+    InstallLocation: APPLICATIONROOTDIRECTORY="<INSTALLPATH>"
+  ProductCode: '{1EC0113B-7DAA-42D2-A089-DEB787FEF1C6}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Tower Deployment Tool
+    DisplayVersion: 8.0.492.0
```
Comment (mdanish-kh) [inline manifests/s/SaaSGroup/Tower/8.0.492/SaaSGroup.Tower.installer.yaml:36]:
```suggestion
```

Not needed. CLI treats `8.0.492` == `8.0.492.0`

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
cc @SpecterShell

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
@stephengillie https://github.com/microsoft/winget-pkgs/issues/152555

**Pull request merged**